.github/workflows/main-branches.yml

```
@@ -26,7 +26,28 @@ jobs:
       - name: Build project
         run: cmake --build .build
 
+  check_tests:
+    needs: [check_compilation]
+    name: Check tests
+    runs-on: ubuntu-latest
+    container: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)